### DISCUSSION TITLE
How do I bulk modify parts?/bulk add image to parts?

### DISCUSSION THREAD

**Discussion by isaacolsen94** [Part-DB/Part-DB-server#565] [Q&A]
How do I bulk modify parts?/bulk add image to parts? I added a bunch or resistors before realizing what I was doing and now want to add an image and a storage location to them but I can't find any way to modify more than one part at a time?

**Comment by jbtronics** [comment]
You can select multiple parts at once in part tables. A new bar will appear over the table then and you can perform operations like delete, changing categories, etc. on them. However that does not work with storage locations or attachments yet, as the meaning of these operations would not be distinct.

**Reply by isaacolsen94** [reply]
Thanks for your quick reply! Is this something that is planned or not currently?
